msbuild/Xamarin.Shared/Xamarin.Shared.props

```diff
@@ -133,6 +133,7 @@ Copyright (C) 2020 Microsoft. All rights reserved.
 		<!-- The default architecture for Xamarin.Mac is x86_64 -->
 		<TargetArchitectures Condition="'$(TargetArchitectures)' == '' And '$(_PlatformName)' == 'macOS'">x86_64</TargetArchitectures>
 		<!-- There should always be an MtouchArch value in newer projects, but for older projects default to old values -->
+		<TargetArchitectures Condition="'$(TargetArchitectures)' == '' And '$(_PlatformName)' == 'tvOS' And '$(ComputedPlatform)' == 'iPhoneSimulator'">x86_64</TargetArchitectures>
```

Comment:
This shouldn't be necessary, all tvOS projects should have an MtouchArch value.

Reply:
I'll try again but I'm pretty sure it did not work. However it might just be the xharness created/cloned projects that are missing the value.